css/myWork.css

```diff
@@ -23,6 +23,180 @@
   animation-duration: 1s;
 }
 
+.popup-main-container {
+  position: relative;
+  padding: 20px;
+  align-items: center;
+  height: 100%;
+  background-color: white;
+  overflow: scroll;
+}
```

Comment:
- [ ] Kindly make sure to remove the horizontal scroll on the popup card in the mobile version in order to match the Figma design.

- [ ] Kindly make sure to reduce the height of the popup card on the mobile version which will help to reduce the empty space at the end of the card.

- [ ] KIndly make sure to add the `border-radius` on the popup card in order to match the provided Figma design.

   - ### YOUR DESIGN
   
   ![image](https://user-images.githubusercontent.com/87197412/224105507-315955d1-acc9-4f95-b566-e551f25dcd11.png)

   - ### FIGMA DESIGN
   ![image](https://user-images.githubusercontent.com/87197412/224105639-a0368ccc-b766-4896-8eb2-a893c5d33432.png)